data/lang/input-core/en.json

```diff
@@ -0,0 +1,203 @@
+    },
+    "BIND_AXIS_THRUST_LEFT": {
+        "message": "Thrust Left / Right",
+        "description": "Descriptive name for the ThrustLeft axis"
+    },
+    "BIND_AXIS_THRUST_UP": {
+        "message": "Thrust Up / Down",
+        "description": "Descriptive name for the ThrustUp axis"
+    },
+    "BIND_THRUST_LOW_POWER": {
+        "message": "Low Power Thrust Mode (hold)",
+        "description": "Descriptive name for the ThrustLowPower action."
+    },
+    "BIND_INCREASE_SPEED": {
+        "message": "Increase Speed Limit",
+        "description": "Descriptive name for the IncreaseSpeed action."
+    },
+    "BIND_DECREASE_SPEED": {
+        "message": "Decrease Speed Limit",
+        "description": "Descriptive name for the DecreaseSpeed action."
+    },
+    "BIND_TOGGLE_SET_SPEED": {
+        "message": "Toggle Speed Control Mode",
+        "description": "Descriptive name for the ToggleSetSpeed action."
+    },
+
+    "BIND_CYCLE_CAMERA_MODE": {
+        "message": "Cycle Camera Mode",
+        "description": "Descriptive name for the CycleCameraMode action."
+    },
+    "BIND_CAMERA_PITCH": {
+        "message": "Camera Pitch",
+        "description": "Descriptive name for the Camera Pitch axis."
+    },
+    "BIND_CAMERA_YAW": {
+        "message": "Camera Yaw",
+        "description": "Descriptive name for the Camera Yaw axis."
+    },
+    "BIND_CAMERA_ROLL": {
+        "message": "Camera Roll",
+        "description": "Descriptive name for the Camera Roll axis"
+    },
+    "BIND_LOOK_PITCH": {
+        "message": "Freelook Pitch",
+        "description": "Descriptive name for the Freelook Pitch axis."
+    },
+    "BIND_LOOK_YAW": {
+        "message": "Freelook Yaw",
+        "description": "Descriptive name for the Freelook Yaw axis."
+    },
+    "BIND_VIEW_ZOOM": {
+        "message": "View Zoom",
+        "description": "Descriptive name for the View Zoom axis"
+    },
+    "BIND_FRONT_CAMERA": {
+        "message": "Switch to Front Camera",
+        "description": "Descriptive name for the FrontCamera action."
+    },
+    "BIND_REAR_CAMERA": {
+        "message": "Switch to Rear Camera",
+        "description": "Descriptive name for the RearCamera action."
+    },
+    "BIND_LEFT_CAMERA": {
+        "message": "Switch to Left Camera",
+        "description": "Descriptive name for the LeftCamera action."
+    },
+    "BIND_RIGHT_CAMERA": {
+        "message": "Switch to Right Camera",
+        "description": "Descriptive name for the RightCamera action."
+    },
+    "BIND_TOP_CAMERA": {
+        "message": "Switch to Top Camera",
+        "description": "Descriptive name for the TopCamera action."
+    },
+    "BIND_BOTTOM_CAMERA": {
+        "message": "Switch to Bottom Camera",
+        "description": "Descriptive name for the BottomCamera action."
+    },
+    "BIND_RESET_CAMERA": {
+        "message": "Reset Camera View to Default",
+        "description": "Descriptive name for the ResetCamera action."
+    },
+
```

Comment:
Normally (as in all other files), we have description field before message field. I think @ecraven added something that will auto-format this when merged (confirm?), so no action should be needed, I think.